.github/workflows/hrm-ecs-reload.yml

```diff
@@ -49,7 +49,6 @@ on:
 
```

Comment:
I left the config reload action region specific because I think it will likely be used that way during new helpline setup.